src/ResourceManager/DataMigration/Commands.DataMigration/help/Stop-AzureRmDataMigrationTask.md

```diff
@@ -0,0 +1,220 @@
+---
+external help file: Microsoft.Azure.Commands.DataMigration.dll-Help.xml
+Module Name: AzureRM.DataMigration
+online version: 
+schema: 2.0.0
+---
+
+# Stop-AzureRmDataMigrationTask
+
+## SYNOPSIS
+Stops an  Azure Database Migration Service task that is in a running state.
+
+## SYNTAX
+
+### ComponentNameParameterSet (Default)
+```
+Stop-AzureRmDataMigrationTask -ResourceGroupName <String> -ServiceName <String> -ProjectName <String>
+ -Name <String> [-PassThru] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+```
+
+### ComponentObjectParameterSet
+```
+Stop-AzureRmDataMigrationTask [-InputObject] <PSProjectTask> [-PassThru]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+```
+
+### ResourceIdParameterSet
+```
+Stop-AzureRmDataMigrationTask [-ResourceId] <String> [-PassThru] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
```

Comment:
@sichoudh looks like a description was missed here. Please make sure that all templates have been removed across markdown files.